front/app/modules/commercial/analytics/admin/hooks/usePostsFeedback/translations.ts

```diff
@@ -0,0 +1,32 @@
+import messages from './messages';
```

Comment:
Replace `utils` by `translations`, seemed cleaner to me

Reply:
yep, cool